package.json

```diff
@@ -45,12 +48,13 @@
     "demo": "npm run build -- --preview",
     "live-demo": "npm run dev",
     "pr": "node scripts/open-pr.js",
+    "pree2e": "npm run clean:e2e",
     "e2e": "wdio wdio.conf.js"
   },
   "husky": {
     "hooks": {
       "pre-commit": "npm run test-suite-inject > /dev/null && pretty-quick --staged",
-      "post-checkout": "npm run leftover-check"
+      "post-checkout": "npm run clean:components && npm run leftover-check"
```

Comment:
Sanity check me here @zhawkins - this makes sense right?  It would be bad to maintain a build from a different branch of code wouldn't it?

Reply:
I think that makes sense. We can try it out for a while and see if it's super annoying. 😄